lib/org/chromium/webidl/CanonicalizerRunner.js

```diff
@@ -32,23 +31,19 @@ foam.CLASS({
       class: 'String',
       name: 'ioRelationshipType',
       documentation: 'The n:m relationship type of input-to-output.',
-      value: 'many:1',
+      value: '*:1',
```

Comment:
I see what you mean. Yes. I think we can leave it like this. The `io` in the name suggests the order in which you would label the one and the many.